content/java/concepts/date/terms/equals/equals.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.equals()'
+Description: 'A method designed to check if the invoking date object
+contains a date and time that is equal to the one specified by date object'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.equals()`** is method of `Date` class. It compares the current date object to the specified object based on millisecond difference. It will returns **`true`** if the invoking date object contains a date that is equal to the one specified by date object. Otherwise, it returns **`false`**.
```

Comment:
```suggestion
The **`.equals()`** method of the `Date` class compares the current `Date` object to the specified object (based on the millisecond difference). It will return `true` if the dates are equal. Otherwise, it returns `false`.
```